fastlane/resources/values/strings.xml

```diff
@@ -3339,6 +3339,7 @@
     <string name="hpp_title">Choose a design</string>
     <string name="hpp_subtitle">Pick your favorite homepage layout. You can edit and customize it later.</string>
     <string name="hpp_choose_button">Choose</string>
+    <string name="hpp_choose_and_create_site">Create site</string>
```

Comment:
I decided to manually update the frozen strings file here to add the new string that landed via #16360 post-code-freeze (and thus initially missed the cut).

Even if the new string landed too late for it to be included in the WeeklyKit which was already sent to our Translation Vendor two days ago, that will at least gives that string a chance to be translated by the polyglots community before end of sprint.